webapp/views/wpt-results.js

```diff
@@ -226,8 +226,13 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
                     </template>
                   </template>
                 </td>
-                <td>
+                <td onclick="[[handleTriageSelect(null, node, testRun)]]" onmouseover="[[handleTriageHover(null, node, testRun)]]">
```

Comment:
I don't think this is quite working on the UI. I can see ` this.handleHover` being triggered when I hover over, but I don't see any css highlights nor will you be able to triage that cell